lightning/src/routing/router.rs

```diff
@@ -1222,7 +1222,7 @@ where L::Target: Logger {
 					let params = ChannelUseParameters {
 						amount_msat: final_value_msat,
 						inflight_htlc_msat: 0,
-						fees_msat: aggregate_next_hops_fee_msat,
+						fees_msat: cmp::max(aggregate_next_hops_fee_msat, aggregate_next_hops_path_htlc_minimum_msat),
```

Comment:
Seeing a couple tests fail that might be related to how this is calculated for last hop hints. At least that's my hunch.

```
thread 'routing::router::tests::our_chans_last_hop_connect_test' panicked at 'assertion failed: `(left == right)`
  left: `4`,
 right: `5`', lightning/src/routing/router.rs:2995:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

thread 'routing::router::tests::prefers_shorter_route_with_higher_fees' panicked at 'assertion failed: `(left == right)`
  left: `100`,
 right: `300`', lightning/src/routing/router.rs:4752:9


failures:
    routing::router::tests::our_chans_last_hop_connect_test
    routing::router::tests::prefers_shorter_route_with_higher_fees

```

Reply:
Hmm... this probably shouldn't cause those failures though.